distributed/worker.py

```diff
@@ -1196,7 +1196,6 @@ async def _register_with_scheduler(self) -> None:
                 middle = (_start + _end) / 2
                 self._update_latency(_end - start)
                 self.scheduler_delay = response["time"] - middle
-                self.status = Status.running
```

Comment:
this extra self.status assignment is redundant because `Server.start` sets the `self.status` to Status.running, but for this assignment the batched_stream hasn't started yet and so `_send_worker_status_change`'s message is dropped silently and so we were relying on `Server.start` doing an extra `Worker.status` transition of `Status.running -> Status.running`